services/terrain/src/terrain/services/admin.clj

```diff
@@ -40,11 +40,14 @@
   [url-to-scrub]
   (str (url/url url-to-scrub :path "/")))
 
+(defn get-with-timeout
+  [url]
+  (client/get url {:socket-timeout 1000 :conn-timeout 1000}))
```

Comment:
Yeah, may as well add an extra 0. What we're trying to prevent is just it running functionally forever/six-hour returns/that sort of thing, it's not like 10 seconds is really an issue even if every request we do as part of this takes that long.